content/blog/2022-08-10-keda-2.8.0-release.md

```diff
@@ -0,0 +1,73 @@
++++
+title = "KEDA 2.8.0 Release"
+date = 2022-08-10
+author = "Jeff Hollan (KEDA), Tom Kerkhove (KEDA)"
+aliases = [
+"/blog/2.8.0-release"
+]
++++
+
+We recently completed our most recent release: 2.8.0 🎉!
+
+Here are some highlights:
+
+- Introduction of new AWS DynomoDB Streams & NATS JetStream scalers.
+- Introduction of new Azure AD Workload Identity authentication provider.
+- Support for specifying `minReplicaCount` in ScaledJob.
+- Support to customize the HPA name.
+- Support for permission segregation when using Azure AD Pod / Workload Identity
+- Additional features to various scalers such as AWS SQS, Azure Pipelines, CPU, GCP Stackdriver, Kafka, Memory, Prometheus
+
+Here are the new deprecation(s) as of this release:
+
+- `rolloutStrategy` in ScaledJob is deprecated in favor of `rollout.strategy`
+
+Let's have a closer look.
+
+## New & Improved Scalers
+
+KEDA now has 50+ built-in scalers, giving you instant access to provide event driven scale across a variety of sources. New with this release are:
+- AWS DynamoDB Streams ([docs](https://keda.sh/docs/2.8/scalers/aws-dynamodb-streams/))
+- NATS JetStream ([docs](https://keda.sh/docs/2.8/scalers/nats-jetstream/))
+
+In addition to new scalers, various new features and improvements were added to AWS SQS, Azure Pipelines, CPU, GCP Stackdriver, Kafka, Memory, and Prometheus scalers.
+
+## Activation and Scaling Thresholds
+
+Previously in KEDA, when scaling from 0 to 1, KEDA would "activate" (scale to 1) a resource when *any* activity happened on that event source. For example, if using a queue, a single message on the queue would trigger activation and scale. We now allow you to set an `activationThreshold` for many scalers which is the metric that must be hit before scaling to 1. This would allow you to delay scaling up to 1 until *n* number of messages were unprocessed. This pairs with other thresholds and target values for scaling from 1 to *n* instances, where the HPA will scale out to *n* instances based on the current event metric and the defined threshold values. Details on thresholds and the new activation thresholds can be found [in the KEDA concept docs](https://keda.sh/docs/2.8/concepts/scaling-deployments/#activating-and-scaling-thresholds)
```

Comment:
Just for readability and easier to digest
```suggestion
Previously in KEDA, when scaling from 0 to 1, KEDA would "activate" (scale to 1) a resource when *any* activity happened on that event source. For example, if using a queue, a single message on the queue would trigger activation and scale.

As of this release, we now allow you to set an `activationThreshold` for many scalers which is the metric that must be hit before scaling to 1.

This would allow you to delay scaling up to 1 until *n* number of messages were unprocessed. This pairs with other thresholds and target values for scaling from 1 to *n* instances, where the HPA will scale out to *n* instances based on the current event metric and the defined threshold values.

Details on thresholds and the new activation thresholds can be found [in the KEDA concept docs](https://keda.sh/docs/2.8/concepts/scaling-deployments/#activating-and-scaling-thresholds)
```